src/header/shared/httpdate.rs

```diff
@@ -46,12 +46,7 @@ impl FromStr for HttpDate {
 
 impl Display for HttpDate {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        let tm = self.0;
-        let tm = match tm.tm_utcoff {
```

Comment:
This was fixed in time crate: https://github.com/rust-lang/time/blob/fba17ed10a4774bb4666c79d58f7cec3ee858865/src/lib.rs#L770